Framework/API/src/TextAxis.cpp

```diff
@@ -28,7 +28,6 @@ Axis *TextAxis::clone(const std::size_t length,
                       const MatrixWorkspace *const parentWorkspace) {
   UNUSED_ARG(parentWorkspace)
   auto newAxis = new TextAxis(*this);
-  newAxis->m_values.clear();
```

Comment:
This change would give inconsistent behavior of `Axis::clone()`, which basically seems to be meant as a polymorphic constructor (*not* copy constructor, i.e., the returned `Axis` is empty). I am not sure this can be changed without causing trouble somewhere. Could you check with someone knowing more about `Axis`, maybe @martyngigg?

Reply:
Yes, this is not critical in fixing the original issue and does make the behavior of `TextAxis` differ from that of other axes. I'll re add this.

Reply:
My recollection is that these methods were designed to be similar to the workspace factory in that they gave an empty object so I think adding this back is the best option.